main/resources/assets/mtsofficialpack/jsondefs/vehicles/comanche.json

```diff
@@ -164,7 +164,7 @@
         {
           "animationType": "rotation",
           "variable": "gear_setpoint",
-          "centerPoint": [0.0,0.75625,0.0],
```

Comment:
Given the new systems, applyAfter is better used here.  Especially since it makes animations easier for new Pack Authors, and the OCP is usually a base example for them.

Reply:
Alright I'll have to have someone else take over this then. I tried applyAfter and just went back to doing the centerpoint since that was the easiest way for me to fix it